go/externals/proof_support_facebook.go

```diff
@@ -159,7 +159,12 @@ func (t FacebookServiceType) NormalizeUsername(s string) (string, error) {
 func (t FacebookServiceType) NormalizeRemoteName(ctx libkb.ProofContext, s string) (string, error) {
```

Comment:
It looks like this function is only called by the `engine/prove.go`. Assertion code calls into `NormalizeUsername` above, and so _will_ normalize away the dots. (So for example, your proofs will still verify even if you and facebook.com disagree about how many dots you have.) However, that's not really obvious from the name -- should we rename it to something else? Or are there maybe use cases I overlooked?

Even in `engine/prove.go`, this change isn't totally clean. `checkExists2` does a string comparison between the output of this function and what's in your ID table, so if you're inconsistent with your own dots, then it will fail to prompt you in the case of an overwrite. We could update that function to call `NormalizeUsername` too? I might do that in a follow-on commit. But this is starting to get a little complicated and I wanted to get your thoughts, @maxtaco.